docs/user/advanced_features.rst

```diff
@@ -1243,6 +1244,50 @@ computation time and complex setup.
 	| Avg Mechanical Power (kW)  |      N/A     |    300      |    241     |
 	+----------------------------+--------------+-------------+------------+
 
+.. _control-reactive-with-PTO:
+
+Reactive Control with Direct Drive Power Take-Off
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The previous controllers only considered the mechanical power output. Although maximization
+of mechanical power allows for the maximum energy transfer from waves to body, it often does 
+not lead to maximum electrical power. The previous controller examples demonstrate the 
+controller types and energy transfer from waves to body, but the important consideration of 
+electrical power requires a PTO model. This example applies a reactive controller to the 
+sphere body with a simplified direct drive PTO model to maximize electrical power. Within 
+the Simulink subsystem for determining the PTO force, the controller prescribes the ideal or 
+desired force which is fed into the direct drive PTO. The current in the generator is then 
+used to control the applied force. 
+
+.. figure:: /_static/images/piPTOSimulink.png
+   :width: 500pt 
+   :align: center
+
+The PTO parameters used for this example are defined in the wecSimInputFile and correspond to 
+the Allied Motion Megaflux Frameless Brushless Torque Motors–MF0310 :cite:`allied`. The 
+results for the applied gains from Section :ref:`control-reactive` are shown in the figure 
```

Comment:
Editorial/minor: Can we add another sentence or two to clarify how the different power levels are calculated. The image piPTOSimulink.png has blue and dashed boxes that are I believe supposed to indicate each step in the power conversion chain; however, clarity on what signals are being used in the calculation would minimize confusion and likely an issue/clarification request. If the piPTOSimulink.png is editable, perhaps putting stars or another indicator on where in the PCC power is being calculated would add clarity.

Reply:
I added a few sentences. I decided not to indicate in the png because the different power levels are combinations of multiple power sources (ex: Mechanical is inertial, damping, torque), so I thought it may cause more confusion.

Reply:
Thanks! Think it is a great addition.